### PR TITLE
Add ut coverage report

### LINKED ISSUE #10405: Integrate coverage report in git hub action for frontend
Since we migrate the unit test from travis to git hub action. It is better to show frontend coverage report there. We need to investigate how to show coverage report.

### PR DESCRIPTION
**Pull request by AllForNothing**
Fix #10405 
Add ut coverage reports provided by codecov for every PR(see below report from a robot account)
The coverage is the sum of ui-ut coverage and backend-ut coverage(can not be provided individually so far). We can see individual reports on detail page

Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -14,16 +14,17 @@ env:
    REDIS_HOST: localhost
    REG_VERSION: v2.7.1-patch-2819-2553
    UI_BUILDER_VERSION: 1.6.0
+   CODECOV_TOKEN: 89c7b95b-dbce-431e-8cb3-55691a9dcc64 #codecov token: visit https://github.com/codecov/codecov-action to learn more
```
Comment (steven-zou) [inline .github/workflows/CI.yml]: It's a plaintext secret

Comment (AllForNothing) [reply .github/workflows/CI.yml]: because I can not think of a better solution

Comment (steven-zou) [reply .github/workflows/CI.yml]: @AllForNothing 

Have you tried other possible ways?

Comment (AllForNothing) [reply .github/workflows/CI.yml]: https://github.community/t5/GitHub-Actions/Make-secrets-available-to-builds-of-forks/m-p/30685#M511    I found the answer here.  It indicate there is no other way so far

File: .github/codecov-ut-config/codecov.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+codecov:
+  require_ci_to_pass: yes
```
Comment (bitsf) [inline .github/codecov-ut-config/codecov.yml]: i think we don't need ci to pass to show this coverage

File: .github/codecov-ut-config/codecov.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```
Comment (bitsf) [inline .github/codecov-ut-config/codecov.yml]: range can be 50 to 100

File: .github/workflows/CI.yml (hunk 1 above)
Comment (bitsf) [reply .github/workflows/CI.yml]: it seems the only risk is hacker may post a fake test coverage report with this token

**Comment by bitsf** [comment]
you should not use fork branch AllForNothing:master to submit pr, it is easily overwritten by mistake

File: .github/codecov-ut-config/codecov.yml (hunk 2 above)
Comment (AllForNothing) [reply .github/codecov-ut-config/codecov.yml]: modified

File: .github/codecov-ut-config/codecov.yml (hunk 3 above)
Comment (AllForNothing) [reply .github/codecov-ut-config/codecov.yml]: modified

**Pull request merged**